src/latbin/lattice-lmrescore-const-arpa.cc

```diff
@@ -25,6 +25,8 @@
 #include "lm/const-arpa-lm.h"
 #include "util/common-utils.h"
 
+#include "lm/kenlm.h"
```

Comment:
Cool.  Might be good idea, since  you're doing this in the same binary, to use an #include guard to optionally disable kenlm support, e.g. #ifndef NO_KENLM ... ?  The idea is that if we have trouble compiling KenLM on some platform, someone can add that to the CXXFLAGS in the Makefile.

Reply:
sure, we will pay extra attention on the install/configure/make system to isolate kenlm as a plugin module.